providers/gitlab.go

```diff
@@ -15,11 +18,55 @@ import (
 type GitLabProvider struct {
 	*ProviderData
 
-	Groups               []string
-	Verifier             *oidc.IDTokenVerifier
+	Groups       []string
+	Projects     []*GitlabProject
+	EmailDomains []string
```

Comment:
It looks like we still have this unneeded `EmailDomains` field in the `GitLabProvider` struct. It is referenced in a unit test as well.

Reply:
Ok, get it, was from previous rewrite of tests to Ginkgo. Removed here https://github.com/oauth2-proxy/oauth2-proxy/pull/630/commits/0658e7fad97fccd45a36d6b2395c2b593700917b